scss/footer.scss

```diff
@@ -0,0 +1,29 @@
+@import 'colors';
+@import 'fonts';
+
+.copyrights {
+  align-self: flex-end;
+  color: white;
+  font-size: $text-small;
+  margin-top: 10px;
+}
+
+.footer-title {
+  color: white;
+  font-size: $text-big;
+  margin-bottom: 10px;
+  text-transform: uppercase;
+}
+
+.footer-container {
+  justify-content: center;
+  align-items: center;
+  background-color: $cerulean;
+  bottom: 0px;
```

Comment:
You shouldn't use bottom /top/left/right with an element that's not `position: absolute`